SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -774,10 +755,27 @@ public void deleteDeadRack(String rackId) {
     return getSingle(requestUri, "slave", slaveId, SingularitySlave.class);
   }
 
+  @Deprecated
   public void decomissionSlave(String slaveId) {
+    decomissionSlave(slaveId, Optional.<SingularityMachineChangeRequest>absent());
+  }
+
+  public void decomissionSlave(String slaveId, Optional<SingularityMachineChangeRequest> machineChangeRequest) {
```

Comment:
updated the new methods, left the old deprecated ones spelled the same since they were already present in the client